src/relayselector.py

```diff
@@ -246,12 +247,14 @@ class UniversalSet(object):
             dictionary of the form { fingerprint : set(...) }.
             """
             def __nonzero__(self): return True
+
             def __contains__(self, obj): return True
             # __len__ is obliged to return a positive integer.
+
```

Comment:
Shouldn't the comment be right above the __len__ method? Even if PEP8 disagrees, I think I would prefer it like that.